### DISCUSSION TITLE
Clear all data in UserDefault when init OpenTelemetry Swift

### DISCUSSION THREAD

**Discussion by Chung14011998** [open-telemetry/community#2267] [Community Discussions]
// Specify the W3C Trace Context, B3, and Jaeger formats to pass trace data.
        let endpoint = URL(string: endPoint)!
        OpenTelemetry.registerPropagators(textPropagators: [W3CTraceContextPropagator()],
                                          baggagePropagator: W3CBaggagePropagator())
        
        // Configuring the dataKey
        let configuration = URLSessionConfiguration.ephemeral
        configuration.httpAdditionalHeaders = ["Authorization": "Bearer token"]
        let session = URLSession(configuration: configuration)
        
        // Setting up attributes
        let attributes: [String: AttributeValue] = [
            ResourceAttributes.serviceName.rawValue: AttributeValue.string("CheckIOS")
        ]
        let resource = Resource(attributes: attributes)
        
        OpenTelemetry.registerTracerProvider(tracerProvider: TracerProviderBuilder()
            .add(spanProcessor: BatchSpanProcessor(spanExporter: OtlpHttpTraceExporter(endpoint: endpoint, useSession: session)))
            .with(resource: resource)
            .build())
        
        _ = URLSessionInstrumentation(configuration: URLSessionInstrumentationConfiguration(shouldInstrument: { _ in return true }))
        
        TraceManager.tracer = OpenTelemetry.instance.tracerProvider.get(instrumentationName: "NSURLSession", instrumentationVersion: "1.0.0")

------------------------------------------------------------------------------------------------------------------------

After I init openTelemetry, all data in UserDefault is removed. How to keep data when init it.
(Only get this error when ios >= 16).
Help me!

**Comment by svrnm** [comment]
@open-telemetry/swift-approvers please take a look.

**Reply by Chung14011998** [reply]
I make a question at https://github.com/open-telemetry/opentelemetry-swift/issues. But not receive any answer
